packages/features/eventtypes/components/EventTypeDescription.tsx

```diff
@@ -40,6 +42,31 @@ export const EventTypeDescription = ({
 
   const paymentAppData = getPaymentAppData(eventType);
 
+  const showManagedBadge = eventType.metadata?.managedEventConfig && !isPublic;
+  const numOfManagedByAdmins = eventType.managedBy?.admins.length || 0;
+  const showManagedByToolTip = showManagedBadge && numOfManagedByAdmins > 0;
+  let managedByToolTipContent = "";
+  let managedByToolTipContentWithLink;
+
+  if (showManagedByToolTip) {
+    managedByToolTipContent = t("managed_by_teamAdmins", {
+      teamAdmins: eventType.managedBy?.admins?.slice(0, 2).join(", "),
+    });
+
+    if (numOfManagedByAdmins > 2 && !!eventType.managedBy?.teamId) {
+      managedByToolTipContentWithLink = (
+        <>
+          {`${managedByToolTipContent} ${t("and")} `}
+          <Link
+            href={`/settings/teams/${eventType.managedBy?.teamId}/members`}
+            className=" text-blue-300 underline">
+            {`${numOfManagedByAdmins - 2} ${t("more")}`}
+          </Link>
+        </>
+      );
+    }
+  }
```

Comment:
Nah, we should show the number of admins if the length is greater than 2. So, remove the check `!!eventType.managedBy?.teamId `and disable the link if teamId is not present.

Reply:
Yeah, but if teamId is null, we wont get the admins as well.
Anyway updated this.
But disabling Link is not possible, as there is no prop for this in <Link